apps/web/examples/phoneInput/phoneInput.root.tsx

```diff
@@ -0,0 +1,52 @@
+import { PhoneInput } from "flowbite-react";
+import type { CodeData } from "~/components/code-demo";
+
+const code = `
+"use client";
+
+import { PhoneInput } from "flowbite-react";
+
+function Component() {
+    return (
+        <form className="max-w-sm mx-auto">
+            <PhoneInput />
+        </form>
+    )
+}
+`;
+
+const codeRSC = `
+function Component() {
+    return (
+        <form className="max-w-sm mx-auto">
+            <PhoneInput />
```

Comment:
not same content

Reply:
for Client & Server?

Reply:
@SutuSebastian Sorry for follow up, awaiting for the clarification for this review.